src/types/config.ts

```diff
@@ -14,11 +14,43 @@ export type OverrideJiterConfigOptions = {
   millisecondsUntilWebhookExpiration?: number;
 };
 
+export type EncryptionOptions<Key = string> = {
+  /**
+   * The encryption keys used to encrypt and decrypt data before sending it to Jiter.
+   *
+   * The last key in the array is the key used to encrypt new data. Old keys are used to decrypt data that was encrypted with that key.
```

Comment:
Can you expand more on how old keys are used? Are they used if decryption fails with a newer key?

Reply:
Yeah, so each key has an `id` and that is sent along with the encrypted payload. So to decrypt the payload you look for the `key` with that `id` and use it to decrypt it. It's not mandatory, but since old data isn't currently removed from the DB the longer a key is used the chance that someone could decrypt that key data would go up the more it's used because they could determine patterns if they had enough data.